src/bundle.ts

```diff
@@ -26,10 +27,10 @@ export const fileOrDirectory = (
         stats.isSymbolicLink()
           ? `(it's a symlink 🔗)`
           : stats.isBlockDevice()
-          ? `(it's a block device 💾)`
-          : stats.isSocket()
-          ? `(it's a socket 🔧)`
-          : `(it's unclear what you have here ❓)`
+            ? `(it's a block device 💾)`
+            : stats.isSocket()
+              ? `(it's a socket 🔧)`
+              : `(it's unclear what you have here ❓)`
```

Comment:
I'm not sure if this is more legible than an if-else, but 🤷‍♂

Reply:
It was, before the new linter got its hands on it. Expect refactoring 🤷‍♂